src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -79,6 +79,10 @@ internal void TestToString(FormulaValue result, StringBuilder sb)
             {
                 sb.Append(n.Value);
             }
+            else if (result is DateValue d)
```

Comment:
Do we have DateTimeValue and TimeValue branches for this as well?

Reply:
I only added what I need to pass existing tests that were failing. I'm guessing there may be needs for DateTimeValue and TimeValue at some point, but I haven't seen tests that require these yet.